static/app/views/releases/list/index.tsx

```diff
@@ -492,11 +527,23 @@ class ReleasesList extends AsyncView<Props, State> {
             {this.renderHealthCta()}
 
             <SortAndFilterWrapper>
-              <SearchBar
-                placeholder={t('Search')}
-                onSearch={this.handleSearch}
-                query={this.getQuery()}
-              />
+              {hasSemver ? (
+                <SmartSearchBar
+                  query={this.getQuery()}
+                  placeholder={t('Search')}
```

Comment:
On Project Details there's `Search by release version` placeholder. Should we have the same one here?